### PR TITLE
Add telegram bot notification.

### PR DESCRIPTION
**Pull request by Teri-anric**
Add telegram bot to notification.
- add create deep link to subscription (get_subscription_link).
- add telegram notification class (TelegramNotifications).
- add bot entrypoint to docker.

### CODE REVIEW

File: web_app/Dockerfile [hunk 1]
```diff
@@ -19,6 +19,7 @@ RUN apt-get update \
     && poetry install --no-interaction --no-ansi
 
 ENTRYPOINT ["bash", "/app/entrypoint.sh"]
+ENTRYPOINT ["bash", "/app/bot_entrypoint.sh"]
```
Comment (djeck1432) [inline web_app/Dockerfile:22]: please add your logic inside `entrypoint.sh`
It could be only one `ENTRYPOINT`

File: web_app/telegram/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from .config import TELEGRAM_TOKEN
+from .handlers import base_router
+from .bot import dp, bot
+from .utils import get_subscription_link, TelegramNotifications
```
Comment (djeck1432) [inline web_app/telegram/__init__.py:4]: Please, before pushing, run `python -m black .` to do formatting

File: web_app/telegram/__main__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from . import dp, bot
+from .middleware import DatabaseMiddleware
+import asyncio
+from database.database import SessionLocal
+
+async def main():
```
Comment (djeck1432) [inline web_app/telegram/__main__.py:6]: Please, change name to describe what is going on. 
For example: `start_pooling` will be better naming then general `main`

File: web_app/telegram/bot.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from aiogram import Bot, Dispatcher
+
+from .config import TELEGRAM_TOKEN
+
+bot = Bot(token=TELEGRAM_TOKEN)
+dp = Dispatcher()
+
+from .handlers import base_router
```
Comment (djeck1432) [inline web_app/telegram/bot.py:8]: Please, move this import to another ones.

File: web_app/telegram/handlers.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+from sqlalchemy.orm import Session
+from sqlalchemy.sql import update
+
+from database.models import NotificationData
+
+base_router = Router()
+
+
+@base_router.message(CommandStart(deep_link=True, deep_link_encoded=True))
+async def start(message: types.Message, db: Session, command: CommandObject):
```
Comment (djeck1432) [inline web_app/telegram/handlers.py:13]: Please, add docstring for better understanding what is inside.

File: web_app/telegram/middleware.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+from aiogram.dispatcher.middlewares.base import BaseMiddleware
+from aiogram.types import TelegramObject
+from sqlalchemy.orm import sessionmaker
+
+class DatabaseMiddleware(BaseMiddleware):
+    def __init__(self, session_pool: sessionmaker):
+        self.session_pool = session_pool
+
+    async def __call__(self, handler: Callable[[TelegramObject, Dict[str, Any]], Awaitable[Any]], event: TelegramObject,
```
Comment (djeck1432) [inline web_app/telegram/middleware.py:11]: Please, add docstring

Comment (djeck1432) [reply web_app/telegram/middleware.py:11]: Add run `python -m black .`

File: web_app/telegram/utils.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+async def get_subscription_link(ident: UUID):
+    me = await bot.me()
+    return create_deep_link(username=me.username, link_type="start", payload=str(ident), encode=True)
+
+
+class TelegramNotifications:
+    __queue_to_send = Queue()
+
+    @classmethod
+    async def send_notification(cls, telegram_id: int):
```
Comment (djeck1432) [inline web_app/telegram/utils.py:18]: add docstring

File: web_app/telegram/utils.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+        await cls.__queue_to_send.put(telegram_id)
+
+    async def __call__(self):
```
Comment (djeck1432) [inline web_app/telegram/utils.py:21]: add docstring

**Comment by Teri-anric** [comment]
Сompleted.
@djeck1432 please check again.

File: web_app/telegram/__main__.py [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import asyncio
+
+from database.database import SessionLocal
+from . import dp, bot
+from .middleware import DatabaseMiddleware
+
+
+async def bot_start_polling():
```
Comment (djeck1432) [inline web_app/telegram/__main__.py:8]: Add please docstring. Every method function should have docstring

**Comment by Teri-anric** [comment]
Okey, @djeck1432 ?

**Review by djeck1432 — APPROVED** [review]

File: web_app/README.md [hunk 10]
```diff
@@ -12,21 +12,40 @@
...
+
 ### 4. Provide all environment varibles needed
 
+```bash
+DB_NAME=#
+DB_USER=#
+DB_PASSWORD=#
+DB_HOST=db
+DB_PORT=#
+TELEGRAM_TOKEN= # token telegram from botfather
```
Comment (Teri-anric) [inline web_app/README.md:38]: Add telegram token to readme.

**Pull request merged**